indico/web/client/styles/design_system/_form_controls.scss

```diff
@@ -0,0 +1,69 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2023 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+@use "utils";
+@use "layout";
```

Comment:
```suggestion
@use 'utils';
@use 'layout';
```

Reply:
not sure why stylelint doen't catch that...

Reply:
Yeah, weird.

Reply:
probably related to that - maybe they already removed it from the standard ruleset: https://stylelint.io/migration-guide/to-15

Reply:
I'll solve the problem on my end.

<img width="239" alt="image" src="https://github.com/indico/indico/assets/65413/e1d0dbe8-d326-4cb7-8c17-05a5352e985e">